src/chromium/crInput.ts

```diff
@@ -33,18 +35,36 @@ function toModifiersMask(modifiers: Set<types.KeyboardModifier>): number {
 }
 
 export class RawKeyboardImpl implements input.RawKeyboard {
-  private _client: CRSession;
+  constructor(
+    private _client: CRSession,
+    private _isMac: boolean,
+  ) { }
```

Comment:
fancy ts!

Reply:
it is a shortcut for:
```ts
class Foo {
  private bar: number;
  constructor(barOpt: number) {
    this.bar = barOpt;
  }
}
```

Reply:
takes an argument in the constructor and immediately sets it as a property. Lets you avoid declaring the type of everything twice and another line to set it.